source/chapters/advanced_topics.rst

```diff
@@ -109,20 +109,55 @@ and `<https://mixxx.org/wiki/doku.php/hid_mapping_format>`_.
 Mixxx Controls
 --------------
 
-See :ref:`controlindex` for a full list.
+Nearly every knob, button, or fader you see in Mixxx's interface is
+controllable via Mixxx's "control" system. The control system allows
+skins, MIDI controllers, and HID controllers to control Mixxx via a
+single interface.
 
-ControlPotMeter controls
-~~~~~~~~~~~~~~~~~~~~~~~~
+A control is identified by a "group" (which is used for grouping
+associated controls) and a "key" (the name of the individual control).
 
+For example, the volume fader for Deck 1 is identified by the group
+`[Channel1]` and key :mixxx:coref:`volume <[Channel1],volume>`. Similarly, the
```

Comment:
why not use :mixxx:coref: for groups, too?
Alos, why

Reply:
I implemented `:mixxx:cogroupref:` instead.